TCMSApp/src/client/app/tests/tests-create.html

```diff
@@ -127,55 +127,45 @@ <h1 class="text-center">Create New Test Case</h1>
 
 <!-- STEPS -->
         <div class="well well-sm">
-          <div class="clearfix">
-            <div class="col-sm-6">
-              <div class="form-group">
-                <label for="step1">Step 1:</label>
-                <textarea class="form-control" rows="5"  class="form-control" id="step1" name="step1" placeholder="Step"></textarea>
+            <div  ng-repeat="s in vmCase.stepList track by $index">
+              <div class="clearfix">
+                <div class="col-sm-6">
+                  <div class="form-group">
+                    <label for="step{{s}}">Step: {{ ::$index + 1 }}</label>
+                    <textarea class="form-control" rows="5"  class="form-control" id="step{{s}}" name="step{{s}}" placeholder="Step"></textarea>
+                  </div>
+                </div>
+                <div class="col-sm-6">
+                  <div class="form-group">
+                    <label for="step{{s}}ExpRes">Expected result:</label>
+                    <textarea class="form-control" rows="5"  class="form-control" id="step{{s}}ExpRes" name="step{{s}}ExpRes" placeholder="Expected result"></textarea>
```

Comment:
you're not using these `id="step{{s}}ExpRes" ` now? will we? :)

and let's have id-s in HTML not in camel case but dash-separated